MAINTAINING.md

```diff
@@ -119,14 +119,21 @@ complete that no input from the original developer or maintainer is required.
 
 ### Organisation of reviewing between maintainers
 
-This section is a placeholder for further discussion around making the review
-process more efficient and effective when multiple maintainers are involved.
-This could include:
-
--   Usage of labels
--   Division of review responsibilities, e.g. with ACKs for different areas of
-    review
--   Usage of GitHub functionality, for example "Reviewers" and "Assignees" lists
+When you review a PR only partly according to the points outlined in the
```

Comment:
Section title? Eg `#### Collaborative reviewing`?

Reply:
Or something that's not basically the same as the whole section. `Partial reviews`?

Reply:
There are no other subsections, so I don't see the point.

Reply:
Ah, I didn't see that `Github etiquette` is already it's own subsection

Reply:
Indeed